cmd/tfsec-skeleton/templates.go

```diff
@@ -54,15 +50,16 @@ func init() {
 }
 `
 
-const checkTestTemplate = `package test
+const checkTestTemplate = `package {{ .Service}}
 
 import (
 	"testing"
 
-	"github.com/aquasecurity/tfsec/internal/app/tfsec/rules"
+	"github.com/aquasecurity/tfsec/internal/app/tfsec/testutil"
 )
 
 func Test_{{.CheckName}}(t *testing.T) {
+	expectedCode := "{{ .ShortCode}}"
```

Comment:
You want `provider-service-shortcode` here

Reply:
Fixed - `{{ .FullCode}}`